css/style.css

```diff
@@ -95,6 +95,20 @@ div.recently-dropped-toggle-container {
 }
 
 /*
+ * ban list
+ */
+
+.ban-source-toggle-container {
+  text-align: right;
+  width: 116px;
+}
+
+.ban-source-toggle-container > ul {
+  text-align: left;
+  width: 150px;
+}
```

Comment:
Yeah I'm not a design person either, so I don't know how or if that could be done.